Paco-Server/src/com/google/sampling/experiential/server/CloudSQLDaoImpl.java

```diff
@@ -448,8 +452,13 @@ public JSONArray getResultSetAsJson(String query, Long eventId) throws SQLExcept
             //if client timezone, then do not write to json
             if (!(colName.equalsIgnoreCase(EventServerColumns.CLIENT_TIME_ZONE))) {
               //if date columns in projection, then display along with corresponding timezone
-              if ((colName.equalsIgnoreCase(EventServerColumns.RESPONSE_TIME)) || (colName.equalsIgnoreCase(EventServerColumns.SCHEDULE_TIME))) {
-                colValue = colValue + rs.getString(EventServerColumns.CLIENT_TIME_ZONE);
+              if(dateColumns.contains(colName)) {
```

Comment:
Adding a note here about line 451 above this since github editor won't let me add a comment on that line. Line 451 contains a potential NullPointerException like we saw recently, rs.getObject(i) may be null in which case calling toString on it would throw an exception.

Reply:
I assumed it was a merging issue. So, did not fix it here. Anyway, now I fixed it.